extensions/arc/src/types.ts

```diff
@@ -19,6 +19,18 @@ export type Space = {
   title?: string;
 };
 
+export type NewTabSearchConfig = {
+  search: string;
+};
+
+export type NewTabSearchConfigs = {
+  google: NewTabSearchConfig;
```

Comment:
curious why use `NewTabSearchConfig` with a single `search` key rather than just `string`?

Reply:
good catch, this was mostly copy pasted from below, then trimmed to only include what was needed.

Updating to use a string.